src/main/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/data/impl/LPGEdgeValue.java

```diff
@@ -0,0 +1,10 @@
+package se.liu.ida.hefquin.engine.wrappers.lpgwrapper.data.impl;
+
+public class LPGEdgeValue {
```

Comment:
I assume you wanted this to be:
```suggestion
public class LPGEdgeValue implements Value {
```